catt/controllers.py

```diff
@@ -95,11 +94,10 @@ def get_stream(url, device_info=None, host=None, ytdl_options=None):
 
 
 def get_app(id_or_name, cast_type=None, strict=False, show_warning=False):
-    arg_is_id = True if re.match("[0-9A-F]{8}$", id_or_name) else False
     try:
-        app = next(a for a in APPS if (a.id if arg_is_id else a.name) == id_or_name)
+        app = next(a for a in APPS if id_or_name in [a.id, a.name])
```

Comment:
A tuple is a slightly more lightweight alternative to the list here, but I'm not fussed either way.